### PR TITLE
Apply Page 

### PR DESCRIPTION
**Pull request by alexh0810**
**What has been done:** 

-  Created apply page (current route is /team/apply) 
- Create a select list with checkboxes with roles as options (roles are hardcoded currently) 
- Add selected roles button should render a list of roles right under the select list 
- Motivation message text input 
- Apply button 

**Current problems:** 
- Add selected roles button should also trigger disabling of the select dropdown list but currently doesn't work 
- Clicking checkboxes also trigger rendering list of roles 
- When ticking checkboxes, duplicate roles are also added 
- Rendered chosen roles should also render description of the roles 

**Future fixes:** 
-Fix current bugs
-Implementation of Tailwind css for components
-Form validation with Formik
-Anything else?

### CODE REVIEW

File: frontend/src/pages/_dashboard/slug/apply/applyForm.js
Comment (wickathou) [inline frontend/src/pages/_dashboard/slug/apply/applyForm.js:1]: Maybe this was not added?

Comment (wickathou) [inline frontend/src/pages/_dashboard/slug/apply/applyForm.js:1]: Maybe this was not added?

File: frontend/src/pages/_dashboard/slug/index.js
Comment (wickathou) [inline frontend/src/pages/_dashboard/slug/index.js:1]: I tried the routes:

`http://localhost:3000/team`
`http://localhost:3000/team/apply`
`http://localhost:3000/dashboard/team`
`http://localhost:3000/dashboard/team/apply`
`http://localhost:3000/dashboard/<event>/team`
`http://localhost:3000/dashboard/<event>/team/apply`

But couldn't load the page, could you share what route works?

**Review by wickathou — CHANGES_REQUESTED** [review]
Hey @alexh0810 

Good work so far! There are some things I comment below for you to check. As I understand, it is all work in progress, so let me know once you are all set for me to review it.

File: frontend/src/pages/_dashboard/slug/apply/applyForm.js
Comment (alexh0810) [reply frontend/src/pages/_dashboard/slug/apply/applyForm.js:1]: It was just an empty file. I'll delete it!

File: frontend/src/pages/_dashboard/slug/index.js
Comment (alexh0810) [reply frontend/src/pages/_dashboard/slug/index.js:1]: Current route for the apply view is (http://localhost:3000/dashboard/<event>/apply)

**Pull request merged**